styles/style.styl

```diff
@@ -220,6 +220,46 @@ section:first-child
   .form--message
     background rgb(226, 33, 112)
 
+.channels
```

Comment:
Would something like this be more consistent with existing styles?

<img width="876" alt="Screenshot 2020-01-14 at 7 56 38" src="https://user-images.githubusercontent.com/13827786/72318028-c91e5980-36a3-11ea-9b38-83f52313ca39.png">

Reply:
Might reduce the strenght/color of the border lines, but some kind of box borders are sensible, because otherwise the visual structure easily gets scattered around. Tried the left aligned channel headings in the beginning, but that wasn't as balanced as the centered one.